src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (firstNumber) => (secondNumber) => {
+    ++count;
```

Comment:
Краще явно збільшувати лічильник
```suggestion
    count += 1;
```